armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -416,6 +422,14 @@ def _checkNDens(self, prevNDen, newNDens, ratio):
                 if prev:
                     self.assertAlmostEqual(prev / new, ratio, msg=f"{prev} / {new}")
 
+    def _checkDetailedNDens(self, prevDetailedNDen, newDetailedNDens, ratio):
```

Comment:
I know this is a private method, but I still think adding a one-line comment would be good.

I know that won't show up in the API docs, but I'm sitting here trying to figure out what this is, and so will the next engineer who has to read this file.

Reply:
added comment